modin/pandas/test/test_general.py

```diff
@@ -437,3 +438,29 @@ def test_to_pandas_indices():
         assert md_df.axes[axis].equal_levels(
             pd_df.axes[axis]
         ), f"Levels of indices at axis {axis} are different!"
+
+
+@pytest.mark.skipif(
+    get_current_backend() != "BaseOnPython",
+    reason="This test make sense only on BaseOnPython backend.",
+)
+@pytest.mark.parametrize(
+    "func, regex",
+    [
+        (lambda df: df.mean(level=0), r"[^D]*DataFrame\.mean"),
+        (lambda df: df + df, r"[^D]*DataFrame\.add"),
```

Comment:
Why you using regexp like `[^D]` ?

Reply:
yeah, you right, `[^D]*` is redundant, removed that